File: Ejercicios_Varios/Java/ProyectoEjerciciosJava/src/main/java/Ejercicios_Condicionales/Ejercicio3.java
package Ejercicios_Condicionales;

import java.util.Scanner;

public class Ejercicio3 {

    public static void main(String[] args) {

        // 3) Declara 2 variables numéricas (con el valor que desees),
        // he indica cual es mayor de los dos. Si son iguales indicarlo también.
        // Ves cambiando los valores para comprobar que funciona.

        Scanner input = new Scanner(System.in);
        System.out.println("Ingrese un primer valor: ");
        System.out.println("Ingrese un primer numero: ");
        int numero1 = input.nextInt();
        System.out.println("Ingrese un segundo numero: ");
        int numero2 = input.nextInt();

        if (numero1 > numero2) {
            System.out.println(numero1 + " es mayor que " + numero2);
        } else if (numero2 > numero1) {
            System.out.println(numero2 + " es mayor que " + numero1);
        } else {
            System.out.println(numero1 + " y " + numero2 + " son iguales");
        }
    }


}

File: Ejercicios_Varios_2/Python/Ejercicios/Ejercicios_Libres/Ejercicio3.py
'''
3) Declara 2 variables numéricas (con el valor que desees),
he indica cual es mayor de los dos. Si son iguales indicarlo también.
Ve cambiando los valores para comprobar que funciona.
'''

numero1= int(input("ingrese un primer valor: "))
numero2= int(input("ingrese un segundo valor: "))

if numero1!= numero2:
    if numero1 > numero2:
        print(f'el primer valor ingresado es mayor al segundo valor')
    else:
        print(f'El segundo valor es mayor que el primer valor ingresado')
else:
    print(f'los valores son iguales, vuelva a intentar')


